main.go

```diff
@@ -250,6 +257,27 @@ func main() {
 	if err != nil {
 		setupLog.Error(err, "could not post Cluster Info to Hub")
 	}
+	sa := &corev1.ServiceAccount{
```

Comment:
@YachikaRalhan I think this complete code should be in seperate function part of hub package.